### DISCUSSION TITLE
Load images posted in comments

### DISCUSSION THREAD

**Discussion by MisterZig** [Docile-Alligator/Infinity-For-Reddit#1477] [Ideas]
Currently when someone posts an image in the comments of a post it just displays the link rather than the image like on the Reddit site.
![Screenshot of the link](https://github.com/Docile-Alligator/Infinity-For-Reddit/assets/107263655/c2ba0301-572e-4659-baa3-93813b2f4fb4)

**Comment by t0stiman** [comment]
Same thing for self-posts

**Comment by ghost** [comment]
I really admire all the hard work and I'm not a dev so I have no idea if this would be a hard function to implement, but it would really increase the user freandliness of the app.
